hopp/simulation/technologies/wind/pysam_financial.py

```diff
@@ -0,0 +1,26 @@
+# tools to add floris to the hybrid simulation class
+from attrs import define, field
+
+import PySAM.Singleowner as Singleowner
+
+from hopp.simulation.base import BaseClass
+from hopp.type_dec import resource_file_converter
+from hopp.utilities.utilities import load_yaml
+
+
+@define
+class PySAMFinancial(BaseClass):
```

Comment:
is this class specific to wind or can we have it live elsewhere?

Reply:
Need to check the input file to see if there are wind specific inputs.

Reply:
PySAM singleowner is be technology agnostic